src/Chapter2.hs

```diff
@@ -336,7 +336,17 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList a b arr
+  | a < 0 || b < 0 = []
+  | a >= 0 && b >= 0 = 
+    let an = max 0 a
+        bn = max (-1) b
+        res1 = take (bn + 1) arr
+        res2 = drop an res1
+    in res2
+
+summ :: Int -> Int -> Int
+summ a b = a+b       
```

Comment:
Is this function used in the module? :thinking:

Reply:
My bad, I forgot to remove this, was debugging something.